README.md

```diff
@@ -93,7 +93,11 @@ Shynet is pretty simple, but there are a few key terms you need to know in order
 
 ## Installation
 
-You can find installation instructions in the [Getting Started Guide](GUIDE.md#installation).
+If you use docker-compose, you can use the docker-compose.yml file to set up shynet and launch it by doing docker-compose up.
+Note that the application will generate a password for you on initial run, which will be output to standard docker logs.
+Also note that this compose file assumes you use a reverse-proxy in front of it, whether Traefik v2, Nginx-Proxy, or a solution on your host.
+
+You can find further installation instructions in the [Getting Started Guide](GUIDE.md#installation).
```

Comment:
I'm going to rephrase this a bit to better fit the way the rest of the documentation is written, but I'll do this on my end before merging.